### PR TITLE
"Authenticator" word to lowercase and of change wording for "mobile application"

### LINKED ISSUE #14314: Login: "authenticator app" should not be capitalized
The word "authenticator", for example in the phrase "authenticator app", is capitalized in a few places. For example:

ranimac3 screenshot 2017-05-22 at 10 36 10

Since "authenticator" is not the name of a specific app, it should not be capitalized. 

On a similiar note, we should probably stick with "authenticator app", and not longer phrases like "authenticator mobile application". For example:

ranimac3 screenshot 2017-05-22 at 10 38 23

### PR DESCRIPTION
**Pull request by iamgabrielma**
Fixes #14314 by changing the "Authenticator" strings into "authenticator" and changing "authenticator mobile application" into "authenticator app"

### CODE REVIEW

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ class VerificationCodeForm extends Component {
 			twoFactorAuthType,
 		} = this.props;
 
-		let helpText = translate( 'Enter the code generated by your Authenticator mobile application.' );
+		let helpText = translate( 'Enter the code generated by your authenticator app.' );
```
Comment (a8ci18n) [inline client/blocks/login/two-factor-authentication/verification-code-form.jsx]: :disappointed: 17 existing translations will be lost with this change.

File: client/me/reauth-required/index.jsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ module.exports = React.createClass( {
 			);
 		} else {
 			codeMessage = this.translate(
-				'Please enter the verification code generated by your Authenticator mobile application.'
+				'Please enter the verification code generated by your authenticator app.'
```
Comment (a8ci18n) [inline client/me/reauth-required/index.jsx]: :disappointed: 26 existing translations will be lost with this change.

File: client/blocks/login/two-factor-authentication/verification-code-form.jsx (hunk 1 above)
Comment (akirk) [reply client/blocks/login/two-factor-authentication/verification-code-form.jsx]: I'll look if we can manage to keep them despite this message :)

File: client/me/reauth-required/index.jsx (hunk 2 above)
Comment (akirk) [reply client/me/reauth-required/index.jsx]: I'll look if we can manage to keep them despite this message :)

**Comment by iamgabrielma** [comment]
Ha, good catch @klimeryk , changed in the last commit.

**Comment by akirk** [comment]
@klimeryk Yes, can be merged. The translations won't be deleted in any case and can be resurrected in case our script misses out on converting them to fuzzy.

File: client/blocks/login/two-factor-authentication/two-factor-actions.jsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ class TwoFactorActions extends Component {
 				{ isAuthenticatorAvailable && (
 					<p>
 						<a href="#" onClick={ this.recordAuthenticatorLinkClick }>
-							{ translate( 'Your Authenticator app' ) }
+							{ translate( 'Your authenticator app' ) }
```
Comment (a8ci18n) [inline client/blocks/login/two-factor-authentication/two-factor-actions.jsx]: :ok: This change is safe, we'll keep the existing 19 translations.

**Comment by klimeryk** [comment]
Nice, thanks for the confirmation, @akirk! 🙇

**Comment by arunsathiya** [comment]
That is one string missing change that I have reported here https://github.com/Automattic/wp-calypso/issues/32654

**Pull request merged**